lib/fizzy/execute.cpp

```diff
@@ -578,9 +579,9 @@ std::unique_ptr<Instance> instantiate(Module module,
     // because table functions will capture the pointer to instance.
     // FIXME: clang-tidy warns about potential memory leak for moving memory (which is in fact
     // safe), but also erroneously points this warning to std::move(table)
-    auto instance = std::make_unique<Instance>(std::move(module), std::move(memory), memory_max,
+    auto instance = std::make_unique<Instance>(std::move(module), std::move(memory), memory_limits,
```

Comment:
No `std::move` for limits, because clang-tidy complains that it's a move of trivial type.